shell/packages/sandstorm-permissions/permissions.js

```diff
@@ -482,6 +474,15 @@ SandstormPermissions.createNewApiToken = function (db, provider, grainId, petnam
         denormalizedGrainMetadata: grainInfo,
       }
     };
+  } else if (owner.grain) {
+    apiToken.owner = {
+      grain: {
+        grainId: owner.grain.grainId,
+        saveLabel: owner.grain.saveLabel,
+        introducerIdentity: owner.grain.introducerIdentity,
+        sealed: owner.grain.sealed,
```

Comment:
whoops, must have missed that.  Fixed in c8e6270ee9734f1b353eae0591f468e15c196354